CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+To contribute to Artipie project you need JDK-11 and Maven 3.2+.
+Some integration tests requires Docker to be installed.
+
+
+## How to contribute
+
+Fork the repository, make changes, and send us a
+[pull request](https://www.yegor256.com/2014/04/15/github-guidelines.html). We will review
```

Comment:
Invalid link for reference. Use this one: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests

Reply:
@Sammers21 the link you have provided just explains what pull request is but not how to use it properly.

Reply:
The only mention of pull requests in the article refers to the exact Github link I sent above:
![image](https://user-images.githubusercontent.com/16746106/87290399-a3fd8d00-c506-11ea-86e9-43f38402ee4b.png)